python/cudnn/grouped_gemm/grouped_gemm_glu/moe_blockscaled_grouped_gemm_glu_rubin.py

```
@@ -0,0 +1,3364 @@
+                (n, k, cutlass.Int32(1)), self.sf_vec_size
+            )
+            sfb = cute.make_tensor(sfb_ptr_typed, sfb_layout)
+
+        # Setup sfa tensor by filling A tensor to scale factor atom layout
+        # ((Atom_M, Rest_M),(Atom_K, Rest_K),RestL)
+        sfa_layout = blockscaled_utils.tile_atom_to_shape_SF(a.shape, self.sf_vec_size)
+        sfa = cute.make_tensor(sfa.iterator, sfa_layout)
+
+        # Setup sfd tensor by filling D tensor to scale factor atom layout
+        self.generate_sfd = sfd_row_tensor is not None and norm_const_tensor is not None
+        if cutlass.const_expr(self.generate_sfd == False):
```

Ruff (0.16.0)

[error] Avoid equality comparisons to `False`; use `not self.generate_sfd:` for false checks

Replace with `not self.generate_sfd`

(E712)